tests/simulation/test_particle_tracker.py

```diff
@@ -648,6 +653,171 @@ def test_particle_tracker_Bethe_warning(
         simulation.run()
 
 
+# def test_particle_tracker_scattering_warning(
```

Comment:
Can we remove this block of commented out tests? Or is this waiting to be put back in?